mypyc/errors.py

```diff
@@ -7,7 +7,7 @@ class Errors:
     def __init__(self) -> None:
         self.num_errors = 0
         self.num_warnings = 0
-        self._errors = mypy.errors.Errors()
+        self._errors = mypy.errors.Errors(hide_error_codes=True)
```

Comment:
Why do we need this change? 🤔

Reply:
The PR would have enabled error codes for `mypyc` as well which resulted in a lot of failing tests. I'm not sure if we should do that too. The main focus was to enable it for `mypy` runs which is why I added `hide_error_codes=True` here.